.pre-commit-config.yaml

```
@@ -8,20 +8,13 @@ repos:
       - id: end-of-file-fixer
       - id: check-yaml
       - id: check-added-large-files
+
   - repo: local
     hooks:
-      - id: nix-fmt
-        name: format nix files
+      - id: treefmt
+        name: Formatting tree
         language: system
         entry: nix fmt
```

```suggestion
        entry: treefmt
```

You can use `treefmt` directly here, as it's available inside the dev shell

---

Ah right, done